docker/docker.go

```diff
@@ -586,7 +671,15 @@ func (d *ImageManager) RunInContainer(opts RunInContainerOpts) (exitCode int, co
 		closeFiles()
 		if err != nil {
 			exitCode = -1
+			return exitCode, container, err
```

Comment:
```suggestion
			return -1, container, err
```

and delete the assignment above, for consistency with other usage in the same function.